sdk/core/azure-core/test/ut/CMakeLists.txt

```diff
@@ -21,8 +21,9 @@ set(CMAKE_CXX_STANDARD_REQUIRED True)
 
 add_executable (
      ${TARGET_NAME}
+     context.cpp
+     datetime.cpp
      http.cpp
-     http.hpp
```

Comment:
?? why

Reply:
Because it is a header file, it is not source, it does not require compiling?

Reply:
Aource files include headers, no need to include headers to compilation.